src/locale/locales/fr/messages.po

```diff
@@ -82,47 +78,47 @@ msgstr "{0, plural, one {Déliker (# like)} other {Déliker (# likes)}}"
 
 #: src/screens/StarterPack/StarterPackScreen.tsx:343
 msgid "{0} people have used this starter pack!"
-msgstr ""
+msgstr "{0} personnes ont utilisé ce pack de démarrage !"
 
 #: src/view/com/util/UserAvatar.tsx:419
 msgid "{0}'s avatar"
 msgstr "Avatar de {0}"
 
 #: src/screens/StarterPack/Wizard/StepDetails.tsx:68
 msgid "{0}'s favorite feeds and people - join me!"
-msgstr ""
+msgstr "Les fils d’actu et les personnes préférées de {0} – rejoignez-moi !"
```

Comment:
Here, I think that `join me` means more "replicate my actions" than "get in this group", so I would rather translate it with this sense in mind (even though `rejoignez-moi` also have this signification in the dictionary — it's less used that way in modern French). 

```suggestion
msgstr "Les fils d’actu et les personnes préférées de {0} – faites comme moi !"
```